test/integration/minishift.go

```diff
@@ -105,6 +110,13 @@ func (m *Minishift) executingMinishiftCommand(command string) error {
 }
 
 func (m *Minishift) getOpenShiftUrl() string {
-	cmdOut, _, _ := m.runner.RunCommand("console --url")
+	cmdOut, _, _ := m.runner.RunCommand(ConsoleURL)
 	return strings.TrimRight(cmdOut, "\n")
 }
+
+func (m *Minishift) getRoutingUrl() string {
+	ocRunner := m.runner.GetOcRunner()
+	cmdOut, _, _ := ocRunner.RunCommand(URLCustomCol)
+	url := strings.Split((cmdOut), "\n")[1]
+	return "http://" + url
```

Comment:
You can get the full url directly via the Minishift command "minishift openshift service ruby-ex -n ruby --url", but of course you need the the application name and project